gravitee-apim-portal-webui-next/src/app/api-details/api-tab-subscriptions/api-tab-subscriptions.component.scss

```diff
@@ -20,3 +20,17 @@
   justify-content: center;
   padding: 20px 0;
 }
+
+.api-tab-subscriptions__table {
+  overflow: hidden;
+  border: 1px solid #e6e0ec;
```

Comment:
Hex codes shouldn't be used directly... reference the palette with a `color-mix`.

Reply:
Done